README.md

```diff
@@ -37,35 +41,35 @@ Since there is no change to the behavior of `require`, there is no change to the
 
 ### Supporting `import` for old-style packages
 
-If a "default.js" file does not exist in the package root, then it will be loaded as an old-style module with no further changes.  It just works.
+If a "default.js" file or "module" main does not exist in the package root, then it will be loaded as an old-style module with no further changes.  It just works.
 
-### Supporting `require` for ES6 packages
+### Supporting `require` for ES Module packages
 
-Since `require` cannot be directly used to import ES6 modules, we need to provide an old-style "index.js" entry point if we want to allow consumers to `require` our package:
+Since `require` cannot be directly used to import ES modules, we need to provide an old-style "index.js" entry point if we want to allow consumers to `require` our package:
 
 ```
 src/
-  [ES6 modules]
+  [ES modules]
 default.js -> src/default.js
 index.js
 ```
 
-The purpose of the "index.js" file will be to map the ES6 module into an old-style module and can be as simple as:
+The purpose of the "index.js" file will be to map the ES module into an old-style module and can be as simple as:
 
 ```js
 // [index.js]
-module.exports = require.import('./src/default.js');
+module.exports = import('./src/default.js');
```

Comment:
I have some additional thoughts here but I'll put them together in a new issue.

Reply:
Yeah this was just illustrative :)